api/src/pcapi/workers/push_notification_job.py

```diff
@@ -31,3 +34,16 @@ def send_offer_link_by_push_job(user_id: int, offer_id: int) -> None:
     offer = Offer.query.get(offer_id)
     notification_data = get_offer_notification_data(user_id, offer)
     send_transactional_notification(notification_data)
+
+
+@job(worker.default_queue)
+def send_unretrieved_bookings_from_offer_notification_job(booking_ids: list[int]) -> None:
+    bookings = bookings_api.get_bookings_with_offers(booking_ids)
+    for booking in bookings:
+        try:
+            notification_data = get_unretrieved_bookings_with_offers_notification_data(booking)
+            send_transactional_notification_delayed(notification_data)
+        except TypeError:
+            logger.error(
+                "booking without an expiration date: could build notification data", extra={"booking": booking.id}
```

Comment:
```suggestion
            logger.exception(
                "booking without an expiration date: could not build notification data", extra={"booking": booking.id}
```

De plus, est-ce que ce `TypeError` peut être levé par `send_transactional_notification_delayed` ? Si non, il faudrait sortir l'appel du `try/catch`.